### ISSUE TITLE
Problem with ansible 2.1

### ISSUE THREAD

**Issue by allamand** [codenrhoden/ansible-role-rexray#4]
Hi,

My Rexray deployment was broken when upgrading to ansible 2.1
In this case it must be a problem with ansible reading playbook configuration and the template is not fulfill anymore, so that it don't works.

I forced install ansible 2.0.1 which I was using previously and then my workflows works again.
http://releases.ansible.com/ansible/ansible-2.0.1.0.tar.gz

I didnt' find what changes in ansible 2.1 but it must be something that broke the actual playbook definition.

**Comment by codenrhoden** [comment]
Good catch -- I haven't tried Ansible 2.1 at all yet.  I'll have to see what isn't backwards compatible.

**Comment by holms** [comment]
This is 2016 issue... is this project abandoned?

**Comment by clintkitson** [comment]
Hello @holms,

REX-Ray adopted the Docker managed plugin model which greatly decreased the reliance on operators to run REX-Ray as a standalone plugin. This project was never brought into the REX-Ray project itself but could if the community is interested in it. Can you describe your usage?

**Comment by holms** [comment]
Just want to have distributed volumes in Docker Swarm via digital ocean spaces. I have I'm actually almost done with my PR wonder it would be accepted here

**Comment by clintkitson** [comment]
Is this something that you'd be willing to answer questions on? If so we
can move it over to the rexray/ github org.

**Comment by holms** [comment]
@clintkitson I'm not sure what you mean. Rexray already supports docker swarm and digital ocean spaces. It's just this playbook doesn't have such configuration implemented somehow

**Comment by clintkitson** [comment]
We are looking for community members to help support features like Ansible
automation. If that is the case then we can justify bringing it into the
REX-Ray project and getting it more visibility.

**Comment by holms** [comment]
@clintkitson Would be nice, because I have at least 5 PR's to make in here in order to use this automation. I can help with reviewing PR's in here, I'm automating with ansible for years already :)
